mayhem/build.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash -eu
```

Comment:
These commands can go in the Dockerfile

Reply:
I am using an OSS-Fuzz base for this submission, so I do not believe this is the case, or at least if it is, it is not conventional.